libbeat/scripts/Makefile

```diff
@@ -31,17 +31,24 @@ GOFILES = $(shell find . -type f -name '*.go')
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "*/vendor/*")
 SHELL=bash
 ES_HOST?="elasticsearch"
+PWD=$(shell pwd)
```

Comment:
I had to readd this line, as without it it breaks. I think the problem is that with the PWD other PWD variable used, during the execution PWD changes. What I need is PWD from the Makefile that is executed.